include/merlin/core_kernels.cuh

```diff
@@ -16,17 +16,7 @@
 
 #pragma once
 
-#include <cooperative_groups.h>
-#include <cooperative_groups/reduce.h>
-#include <cuda/barrier>
-#include <mutex>
-#include <thread>
-#include <vector>
-#include "types.cuh"
-#include "utils.cuh"
-
-using namespace cooperative_groups;
-namespace cg = cooperative_groups;
+#include "core_kernels/kernel_utils.cuh"
```

Comment:
Please modify for Bazel build file at the same time, its location is `./include/merlin/BUILD`, and please try to build with Bazel after done.